examples/grpc-client-client-streaming/grpc_client_streaming.md

```diff
@@ -1,6 +1,6 @@
 # gRPC client - Client-side streaming RPC
 
-The gRPC Server Connector is used to expose gRPC services over HTTP/2. In a gRPC client streaming scenario, the client writes a sequence of messages and sends them to the server. Once the client has finished writing the messages, it waits for the server to read them and return a response.
+The `grpc:Client` communicates with a gRPC server over HTTP2. A `grpc:Client` is created by providing the endpoint url of gRPC server. In the client streaming scenario, once connected, the client sends a sequence of messages to the remote service and waits for the server to read them and return a single response. Use this to send multiple requests and get a single response back.
```

Comment:
endpoint URL

Reply:
Updated in https://github.com/ballerina-platform/ballerina-distribution/pull/3785/commits/58b1668c71f383fe61fcd9fd3f59ded060eee2fd